firestore-translate-text/extension.yaml

```diff
@@ -156,3 +166,26 @@ params:
     example: translated
     default: translated
     required: true
+
+  - param: DO_BACKFILL
+    label: Translate existing documents
+    description: >
+      Should existing, untranslated documents in the Firestore collection be translated as well?
+    type: select
+    options:
+      - label: Yes
+        value: true
+      - label: No
+        value: false
+
+
+lifecycleEvents:
+  onInstall:
+    function: fstranslatebackfill
```

Comment:
I'm not sure if we have the required params at this stage?

Display shows `Translating existing documents in ${param:COLLECTION_PATH}`

Attempted with:

 ${COLLECTION_PATH}
 ${param:COLLECTION_PATH}

This could be a deploy from local issue

Reply:
Ahhh, I forgot that we show these preinstall as well. Fixing

Reply:
Confirmed that we aren't subbing this correctly at the moment. @ifielker is working on a backend fix to do so

Reply:
Fix should land on Monday. Sorry about that.

Reply:
Seems to be working now. Thanks!